core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/NoticeExport.java

```diff
@@ -0,0 +1,7 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+/**
+ * Annotation to be used on notice constructor. This specifies the constructor to be considered
+ * while exporting notice information.
+ */
+public @interface NoticeExport {}
```

Comment:
@lionel-nj One comment here - I would suggest renaming this annotation and providing a more detailed description regarding the exact use case for which it exists.

For example, I think `SchemaExport` may be a better name here, or `NoticeSchema`, or something with `schema` in it, to distinguish it from the export of the notices themselves. It's not clear by the name or description of the annotation what exactly the annotation is used for - I'd link to the class that processes it too when that exists.